src/Agent.Worker/Build/GitSourceProvider.cs

```diff
@@ -1114,6 +1114,13 @@ public override async Task RunMaintenanceOperations(IExecutionContext executionC
                     throw new InvalidOperationException($"Git prune failed with exit code: {exitCode_prune}");
                 }
 
+                // git lfs prune
+                int exitCode_lFSPrune = await _gitCommandManager.GitLFSPrune(executionContext, repositoryPath);
```

Comment:
Could it make sense to check that lfs is installed before running lfs prune command?

Reply:
Thank you for the review.  I added a check using GitLfsVersion.  Unfortunately the x64 pipeline jobs [have started failing](https://dev.azure.com/mseng/PipelineTools/_build/results?buildId=14524575&view=logs&jobId=f4485888-349a-5114-5431-db0c31b0a7f9&j=f4485888-349a-5114-5431-db0c31b0a7f9&t=ff404933-910b-51c1-ef3d-757efe5ecd33) in the `Component Detection` stage.

I do not think those failures are related to the changes I propose.